ballerina/admin.bal

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) 2023 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
```

Comment:
```suggestion
// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org).
//
// WSO2 LLC. licenses this file to you under the Apache License,
// Version 2.0 (the "License"); you may not use this file except
// in compliance with the License.
// You may obtain a copy of the License at
```

Reply:
Apply the same for all the newly added file headers

Reply:
Addressed in [c897e7d](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/180/commits/c897e7d12cbfc057afc2c3f16842feb00b5e2abe)